app/routes.py

```diff
@@ -1 +1,267 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
+from .models.task import Task
+from .models.goal import Goal
+from sqlalchemy import asc, desc
+from datetime import datetime
+from flask import Blueprint, jsonify, make_response, request, abort
+import requests
+import os
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    
+    if request_body.get("title") is None or request_body.get("description") is None:
+        return make_response ({"details": "Invalid data"}, 400)
+    
+    
+    new_task = Task.from_dict(request_body)
+
+    db.session.add(new_task)
+    db.session.commit()
+
+    response = {"task": new_task.to_dict()}
+
+    return make_response(jsonify(response), 201)
+
+@tasks_bp.route("", methods=["GET"])
+def get_all_tasks():
+
+    order_param = request.args.get("sort")
+
+    
+    if order_param == "asc" :
+        tasks = Task.query.order_by(Task.title.asc())
+    elif order_param == "desc": 
+        tasks = Task.query.order_by(Task.title.desc())
```

Comment:
Interesting, I was under the impression that you'd need `.all()` at the end of the query in order for SQL Alchemy to return a list. But the fact that you're later iterating over tasks, suggests to me that an iterable is returned, so I'd be curious to see what `tasks` looks like when `.all()` isn't called vs when it is.